content/benefits/education-benefits/release-notes/2023-06-09.md

```diff
@@ -0,0 +1,19 @@
+We've added the new Education Benefits API to the developer portal. It replaces the EVSS Education Service. The Education Benefits API returns information about a Veteran’s education benefits, including:
+
+
+- Whether a Veteran is eligible for the Post-9/11 GI Bill’s education benefits.
+- The total education benefits earned by the Veteran.
+- The amount of a Veteran’s education benefits that has been used.
+- The remaining amount of a Veteran’s education benefit.
+- Details of educational enrollments for which benefits were used, including start and end dates, schools, and credits received. 
+
+
+
+This API uses the client credentials grant (CCG) flow to provide an access token to trusted systems. See our [CCG guide](https://developer.va.gov/explore/authorization/docs/client-credentials?api=claims) for more details. 
+
+- **Important**: To get production access for this API, you must work for VA or have specific VA agreements. If you have any questions, [contact us](https://developer.va.gov/support/contact-us).
+
+As always, it's important to know that the new version must support backward compatibility. It should:
+- Be tolerant of unknown fields in the payload. Ignore the deserialization of new fields but do not eliminate them from the payload if needed for subsequent PUT requests. 
```

Comment:
```suggestion
- Be tolerant of unknown fields in the payload. You should ignore the deserialization of new fields, but don't eliminate them from the payload in case they are needed for subsequent PUT requests. 
```